### ISSUE TITLE
BSIconButton has no ButtonCustomization added to it.

### ISSUE THREAD

**Issue by ShaHar91** [stevdza-san/KotlinBootstrap#9]
By using the BSButton I noticed that I can create custom shapes and colors for a specific button. 
For the BSIconButton I would assume the same behaviour but that's not available at the moment.

Is this something that got overlooked or just something that's not on the roadmap?

**Comment by stevdza-san** [comment]
Yes, that's correct, I still haven't enabled the customization for all components. I'm planning to add more customization options, but can't say the exact date/time yet.
